Misc/NEWS.d/next/Core and Builtins/2020-09-14-09-15-59.bpo-1635741.UnXTND.rst

```diff
@@ -0,0 +1,2 @@
+Convert the getstats method of the :mod:`_lsprof` extension module to
+argument clinic

```

Comment:
Please remove the NEWS entry, it's not worth it.

Reply:
@vstinner done